Dockerfile

```
@@ -0,0 +1,12 @@
+FROM python:3.7-alpine
+
+WORKDIR /px-ldap-replace
```

According to FHS this needs to live in `/opt/`.
```suggestion
WORKDIR /opt/px-ldap-replace
```